### ISSUE TITLE
Choose a build tool

### ISSUE THREAD

**Issue by rooftopsparrow** [rooftopsparrow/resource-locking#6]
Current proposed tools are:

- Maven
- Gradle
- Ant

It is also possible that we don't need a build tool at all, and we all use the same IDE or something. Lets discuss and come to a consensus.

**Comment by hdemarco4** [comment]
I've been using IntelliJ.  I know it has all of those built in, but I haven't used any of them on their own.

**Comment by Mojac** [comment]
I've just been using jGRASP all these years with an occasional text editor (vi or Sublime Text 3). I could download IntelliJ or other IDE and use those. I haven't really looking into any so I'm up for using anything you guys recommend.

**Comment by arcturas1** [comment]
I vote Gradle, as I've not used any particular build tool before (aside from whatever Eclipse uses by default), but it looks good based on the discussion at Monday's class meeting, and I'm down to learn it.

@Mojac  - your IDE is independent of the built tool you use. Many IDE's support integration with build tools such as Maven, Gradle, and Ant, but if you choose to use an IDE like Eclipse or IntelliJ, it just needs to be configured to use whichever tool we choose here. As to IDE recommendations, I personally use and greatly enjoy Eclipse, and it does come with built-in support for all 3 of the proposed build tools. Eclipse can be a bit testy at times, so be prepared to Google how to configure certain things, but it's been a great tool for me, and there's lots of documentation for it so it's pretty easy to learn how to make it behave as you like.

**Comment by rooftopsparrow** [comment]
I think we have a majority for gradle. Unless anyone is 100% blocked by it, lets move forward with it. I'll be creating a new issue and linking here for setting up gradle.
